tests/Ninja/Build/absolute-dependencies.ninja

```diff
@@ -0,0 +1,31 @@
+# Check that touching absolute transitive dependencies affect relative targets.
```

Comment:
We should probably include a note *somewhere* that we are striving to be "more correct" than the actual Ninja, in this regard, since it is otherwise a little confusing if anyone ever wonders why.